docs/common/input.md

```diff
@@ -401,6 +401,11 @@ RDycore. The parameters that define these discretizations are
 ```yaml
 output:
   directory: output
+  fields:
```

Comment:
How about updating `Houston1km.DirichletBC.yaml` instead of this one, as we will get non-zero values for `WaterSource`?